backend/app/services/knowledge/indexing.py

```
@@ -369,6 +369,46 @@ def run_document_indexing(
+                        f"[Indexing] Deleted old index before re-indexing: document_id={document_id}, "
+                        f"deleted_chunks={delete_result.get('deleted_chunks', 0)}"
+                    )
+                    add_span_event(
+                        "rag.indexing.old_index_deleted",
+                        {
+                            "kb_id": str(knowledge_base_id),
+                            "document_id": str(document_id),
+                            "deleted_chunks": delete_result.get("deleted_chunks", 0),
+                        },
+                    )
+                except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)